shuffle/cli/src/shared.rs

```diff
@@ -407,44 +414,59 @@ pub struct NetworksConfig {
 }
 
 impl NetworksConfig {
-    pub fn default() -> Self {
+    pub fn default() -> Result<Self> {
         let mut network_map = BTreeMap::new();
-        network_map.insert("localhost".to_string(), Network::default());
-        NetworksConfig {
+        network_map.insert(LOCALHOST_NAME.to_string(), Network::localhost()?);
+        network_map.insert(TROVE_TESTNET_NAME.to_string(), Network::trove_testnet()?);
+        Ok(NetworksConfig {
             networks: network_map,
-        }
+        })
     }
 
-    pub fn url_for(&self, network_name: &str) -> Result<Url> {
-        let specified_network = self.networks.get(network_name).ok_or_else(|| {
+    pub fn get_network(&self, network_name: &str) -> Result<Network> {
```

Comment:
nit: What's wrong with just `get()`? What else would we get?

Reply:
True